presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/TopologyAwareNodeSelector.java

```diff
@@ -69,7 +69,7 @@ public class TopologyAwareNodeSelector
     private final NodeSelectionStats nodeSelectionStats;
     private final NodeTaskMap nodeTaskMap;
     private final boolean includeCoordinator;
-    private final AtomicReference<Supplier<NodeMap>> nodeMap;
+    private final AtomicReference<Supplier<NodeSet>> nodeMap;
```

Comment:
Idk, I am not sure why we are going with this change from NodeMap to NodeSet, it is increasing the diff size.